File: src/main/java/com/example/climbingapi/model/UserRoute.java
package com.example.climbingapi.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.OffsetDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UserRoute {

    private Integer id;
    private Integer userId;
    private Integer routeId;
    private OffsetDateTime tickedAt;
    private String style;
    private Integer rating;
    private String personalNote;
}

File: src/main/kotlin/com/example/climbingapi/model/UserRoute.kt
package com.example.climbingapi.model

import java.time.OffsetDateTime

data class UserRoute(
    val id: Int?,
    val userId: Int?,
    val routeId: Int?,
    val tickedAt: OffsetDateTime?,
    val style: String?,
    val rating: Int?,
    val personalNote: String?
)
